src/examples/java/io/nats/examples/NatsJsPub.java

```diff
@@ -28,7 +28,7 @@
 
 public class NatsJsPub {
 
-    static final String usageString = "\nUsage: java NatsJsPub [-s server] [-h headerKey:headerValue]* <subject> <message>\n"
+    static final String usageString = "\nUsage: java NatsJsPub [-s server] [--stream name] [-h headerKey:headerValue]* <subject> <message>\n"
```

Comment:
add stream name to the other examples

Reply:
done, and added stream creation in the example subs so they can be run first.